AuxTel/Troubleshooting/ATCS/AuxTel-Dome-Fails-to-Arrive-in-Position.rst

```diff
@@ -0,0 +1,187 @@
+    await self.arun(checkpoint=True)
+  File "/home/saluser/ts_externalscripts/python/lsst/ts/externalscripts/auxtel/latiss_acquire_and_take_sequence.py", line 731, in arun
+    await self.latiss_acquire()
+  File "/home/saluser/ts_externalscripts/python/lsst/ts/externalscripts/auxtel/latiss_acquire_and_take_sequence.py", line 484, in latiss_acquire
+    tmp, data = await asyncio.gather(
+  File "/home/saluser/ts_observatory_control/python/lsst/ts/observatory/control/base_tcs.py", line 659, in slew_icrs
+    raise slew_exception
+  File "/home/saluser/ts_observatory_control/python/lsst/ts/observatory/control/base_tcs.py", line 619, in slew_icrs
+    await self.slew(
+  File "/home/saluser/ts_observatory_control/python/lsst/ts/observatory/control/base_tcs.py", line 807, in slew
+    await self._slew_to(
+  File "/home/saluser/ts_observatory_control/python/lsst/ts/observatory/control/auxtel/atcs.py", line 1447, in _slew_to
+    await self.process_as_completed(self.scheduled_coro)
+  File "/home/saluser/ts_observatory_control/python/lsst/ts/observatory/control/remote_group.py", line 1173, in process_as_completed
+    raise e
+  File "/home/saluser/ts_observatory_control/python/lsst/ts/observatory/control/remote_group.py", line 1170, in process_as_completed
+    ret_val = await res
+  File "/opt/lsst/software/stack/conda/envs/lsst-scipipe-5.1.0/lib/python3.10/asyncio/tasks.py", line 571, in _wait_for_one
+    return f.result()  # May raise f.exception().
+  File "/home/saluser/ts_observatory_control/python/lsst/ts/observatory/control/auxtel/atcs.py", line 1630, in wait_for_inposition
+    status = await asyncio.gather(*tasks)
+  File "/home/saluser/ts_observatory_control/python/lsst/ts/observatory/control/auxtel/atcs.py", line 1696, in wait_for_atdome_inposition
+    await asyncio.wait_for(self.dome_az_in_position.wait(), timeout=timeout)
+  File "/opt/lsst/software/stack/conda/envs/lsst-scipipe-5.1.0/lib/python3.10/asyncio/tasks.py", line 458, in wait_for
+    raise exceptions.TimeoutError() from exc
+  asyncio.exceptions.TimeoutError
+
+
+
+.. _AuxTel-Dome-Fails-to-Arrive-in-Position-Solution:
+
+Solution
+=========
+
+The dome tracking and positioning can be recovered from LOVE. 
+In case this fails, you can use a notebook to issue commands an recover the dome position as following. 
+All of the commands you will need can be found in the daytime_checkout notebook.
+
+
+A. **Procedure - Recovery from LOVE.**
+
+   1. State cycle ATDome through ``STANDBY`` and back to ``ENABLE`` from the ASummaryState. 
+      The transition must be quick enough so the dome shutter does not start closing and the recovery is faster; 
+      if it does close, the next step should deal with the shutter opening again.  
+      ``FAULT`` → ``STANDBY`` → ``START`` → ``DISABLED`` → ``ENABLED``.
+   
+      a. If ATPtg faulted, transition it back to enabled from ASummaryState. 
+
+   2. Select `auxtel/prepare_for/onsky` and move it to the beginning of the ATQueue. 
```

Comment:
I suggest for readability :
.... Select `auxtel/prepare_for/onsky` from the script list and move ...